src/middlewared/middlewared/etc_files/ctld.py

```diff
@@ -1,35 +1,16 @@
 #!/usr/local/bin/python
-from middlewared.client import Client
 from middlewared.client.utils import Struct
+import contextlib
 import logging
-import logging.config
+import logging.handlers
 import os
+import sysctl
 
+handler = logging.handlers.SysLogHandler()
+handler.setLevel(logging.DEBUG)
+handler.setFormatter(logging.Formatter('[%(name)s:%(lineno)s] %(message)s'))
```

Comment:
Curious, wont this set the logging handler to the middlewared process instead of have this contained to this alone?

Perhaps we should just log to middlewared.log now?

Reply:
@william-gr it only adds this handler to `logging.getLogger('clt.conf.py')` so yes, handler is kept for entire process, but only for `clt.conf.py`

I can log to middleware instead if no one relies on this message being in syslog

Reply:
Reason it was in syslog was because it had to be logged somewhere, since it was previously a standalone process and users could not get notified otherwise. I think it makes sense to use our existing logging facilities here